pkg/operator/operator_test.go

```diff
@@ -0,0 +1,131 @@
+package operator
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/certificates/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+const dir = "_tmp"
```

Comment:
I'd generally suggest moving this into the test func to not pollute the identifier space of the whole package.

But actually, here you may rather want to us this stdlib function: https://pkg.go.dev/io/ioutil#TempDir